testnets/mantrachaintestnet/chain.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mantrachaintestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "MANTRA Chain",
+  "chain_id": "mantrachain-testnet-1",
+  "bech32_prefix": "uaum",
+  "daemon_name": "mantrachaind",
+  "node_home": "$HOME/.mantrachain",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uaum",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.003
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uaum"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/MANTRA-Finance",
+    "recommended_version": "1.0.0",
+    "compatible_versions": [
+      "1.0.0"
+    ],
+    "cosmos_sdk_version": "0.47.3",
+    "consensus": {
+      "type": "cometbft",
+      "version": "0.37"
+    },
+    "cosmwasm_version": "0.41",
+    "cosmwasm_enabled": true,
+    "genesis": {
+      "genesis_url": ""
+    },
+    "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ],
+        "cosmos_sdk_version": "0.47.3",
+        "consensus": {
+          "type": "cometbft",
+          "version": "0.37"
+        },
+        "cosmwasm_version": "0.41",
+        "cosmwasm_enabled": true
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "69fa5f7927f2b013f152b6dfc56324156feb1973",
+        "address": "34.172.80.207:26656",
+        "provider": "MANTRACHAIN"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "182a37a556ff0b6733fe58020e7746de3292492d",
+        "address": "5.222.198.102:26656",
+        "provider": "MANTRACHAIN"
+      },
+      {
+        "id": "64691a4202c1ad29a416b21ce21bfc9659783406",
+        "address": "34.136.169.18:26656",
+        "provider": "MANTRACHAIN"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc.testnet.mantrachain.io",
+        "provider": "MANTRACHAIN"
+      },
```

Comment:
this comma makes the schema validation fail

Reply:
Fixed